user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Feeds/Specifying_data_input_in_a_dashboard_URL.md

```diff
@@ -35,7 +37,12 @@ This JSON object has to have the following structure:
 ```
 
 > [!TIP]
-> See [Example of passing the data using a JSON object in the URL](#example-passing-the-data-using-a-json-object-in-the-url)
+> See [Examples of passing the data using a JSON object in the URL](#examples-passing-the-data-using-a-json-object-in-the-url)
+
+- The version number is currently always 1.
+
+  > [!IMPORTANT]
+  > In a low-code app URL, use "v" instead of "version" to specify the version number.
```

Comment:
This is not correct. Version is also valid. Actually the version is currently ignored because we don't have any different versions yet. Omitting the "version" will still work. It becomes important when this json object would change in the future version and would need a migration.

Reply:
```suggestion
  > In a low-code app URL, both "v" and "version" can be used to specify the version number.
```

I added this note because I noticed "v" was used in the low-code app example further down. Is this correction accurate?

Reply:
the example is wrong, "version" does not get encoded to %22v%22. I checked the code and only "version" is used to write to the url, and currently not yet used to read from the url. So "version" is the only valid (currently ignored) syntax for both dashboards and LCA.